### PR TITLE
Update Leap 2.2.7 & Bugfix

### PR DESCRIPTION
**Pull request by RandomOutput**
**Changes:**
- Update Leap PC and Mac DLLs and Bundles to 2.2.7
- Update Mac Quick Switch build to 2.2.7
- Fix bug in VR scenes where there were missing references in alignment system.
- Fix bug where _imageList could be accessed when null in LeapImageRetreiver

**To Test:**
- Run all VR scenes in editor and confirm they exhibit proper functionality (what the scene is meant to communicate or do) and no errors are thrown in the editor: 
 - [x] 500 Blocks
 - [x]  Hand Viewer
 - [x]  Moving Demo
 - [x]  Quick Switch Demo
 - [x] Space Time Alignment
 - [x] VR Widgets 
- Confirm that Leap OVR Camera prefabs produce reasonable, working results when added to a scene:
 - [x] LeapOVRCameraRig
 - [x] LeapOVRPlayerController
 - [x] QuickSwitchCameraRig

### CODE REVIEW

**Comment by Amarcolina** [comment]
Looks like you might have an empty folder, there is a .meta file for Assets/OVR/Fonts but the folder doesn't exist when I pull

**Comment by Amarcolina** [comment]
In VR Widgets I fell through the floor.  Looks like the player controller doesn't have anything to stand on.  Also possibly a scene that should just use the camera rig instead of the player controller?

**Comment by RandomOutput** [comment]
mist have broken something when I updated the prefabs. Will investigate.

**Comment by RandomOutput** [comment]
@Amarcolina Issues you reported should be resolved.

File: Assets/LeapMotion+OVR/Scripts/LeapCameraAlignment.cs [hunk 1]
```diff
@@ -229,6 +237,7 @@ protected struct TransformData {
   }
 
   void UpdateAlignment () {
+    Debug.Log(deviceInfo.type);
```
Comment (Amarcolina) [inline Assets/LeapMotion+OVR/Scripts/LeapCameraAlignment.cs]: This should probably either be removed or moved into verbose debugging

**Comment by Amarcolina** [comment]
Everything looks good besides that one debug that is being spammed

File: Assets/LeapMotion+OVR/Scripts/LeapCameraAlignment.cs (hunk 1 above)
Comment (RandomOutput) [inline Assets/LeapMotion+OVR/Scripts/LeapCameraAlignment.cs]: Whoops! Thought I deleted that. Must not have committed it.

**Pull request merged**